helpers/swagger_security_handlers.js

```diff
@@ -0,0 +1,5 @@
+
+
+module.exports = {
+    
+};
```

Comment:
Is this place holder required?

Reply:
Its a placeholder to move code of `httpApi.middleware.applyAPIAccessRules` to swagger pipe stack.